Example oas file is failing some aep-158 linter checks

Specifically:

```text
>spectral lint examples/example.oas.yaml 
(node:3533) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

/Users/Mike/Projects/aep-dev/aep-openapi-linter/examples/example.oas.yaml
 44:48  error  aep-158-response-next-page-token-property  The response schema must include a next_page_token property.  paths./isbns.get.responses[200].content.application/json.schema.properties

✖ 1 problem (1 error, 0 warnings, 0 infos, 0 hints)
```

--- a/aep/general/example.oas.yaml
+++ b/aep/general/example.oas.yaml
@@ -43,6 +43,7 @@
                                   'type': 'object',
                                   'properties':
                                     {
+                                      'next_page_token': { 'type': 'string' },
                                       'results':
                                         {
                                           'type': 'array',
@@ -171,6 +172,7 @@
                                   'type': 'object',
                                   'properties':
                                     {
+                                      'next_page_token': { 'type': 'string' },
                                       'results':
                                         {
                                           'type': 'array',
@@ -407,6 +409,7 @@
                                   'type': 'object',
                                   'properties':
                                     {
+                                      'next_page_token': { 'type': 'string' },
                                       'results':
                                         {
                                           'type': 'array',
@@ -675,6 +678,7 @@
                                   'type': 'object',
                                   'properties':
                                     {
+                                      'next_page_token': { 'type': 'string' },
                                       'results':
                                         {
                                           'type': 'array',

